components/profile/sidebarcard/README.md

```diff
@@ -0,0 +1,52 @@
+# ProfileSidebarcard
```

Comment:
Since we already have a `card` context defined in `sui-components` i ould define this component as `card/profile`.
In my opinion, we should not reference the location where it's gonna be placed (Sidebar), becasuse we don't know where will we use it in the future.

Reply:
Done, renamed the component